feed/services.py

```
@@ -0,0 +1,18 @@
+from django.contrib.contenttypes.models import ContentType
+
+from feed.constants import SIGNALS_MODELS
+from news.models import News
+
+
+def delete_news_for_model(instance: SIGNALS_MODELS):
+    content_type = ContentType.objects.get_for_model(instance)
+    obj = News.objects.filter(content_type=content_type, object_id=instance.id).first()
```

.first() не кинет исключение если ничего не найдется в базе?

---

Django 1.6  introduced the first() and last() which swallow the resulting exception and return None if the queryset returns no objects.

https://stackoverflow.com/questions/5123839/fastest-way-to-get-the-first-object-from-a-queryset-in-django

---

кайф, не знал)